src/App.tsx

```diff
@@ -161,7 +161,7 @@ function App() {
   const endDate = startEnd.start ? 
     <div className="pair">
       <label htmlFor="end">Vacation End</label>
-      <input type="date" name="end" id="end" defaultValue={`${startEnd.start.toISOString().slice(0,10)}`} onChange={handleStartEndChange}/>
+      <input autoFocus type="date" min={new Date(startEnd.start).toISOString().split('T')[0]} name="end" id="end" defaultValue={`${startEnd.start.toISOString().slice(0,10)}`} onChange={handleStartEndChange}/>
```

Comment:
I think this would greatly benefit from being pulled out into a const higher up in the component.

```suggestion
       const startsAt = new Date(startEnd.start).toISOString()
       
       ... 
       
      <input autoFocus type="date" min={startsAt.split('T')[0]} name="end" id="end" defaultValue={`${startsAt.slice(0,10)}`} onChange={handleStartEndChange}/>
```

A few other questions:
1. What is `startEnd.start`? I would have assumed it was some kind of ISO string already, so passing it to Date just to turn it back into an ISO string feels odd to me.
2. Is <input> not able to just accept an ISO string? I'm curious why we are slicing and dicing the string before passing it into `min` and `defaultValue`.

Reply:
Thanks for taking a look!

1) `startEnd.start` is just the start date for the vacation date range. I do agree that changing it to a date and then back to ISOstring feels weird; I remember it being buggy when I didn't convert it to a date first for some reason before passing it into this input element. I'll play around with it more and see if I can figure out if I can avoid this double conversion!

2) I was slicing the string to just get the date part, I guess both the `min` and `defaultValue` are slicing the exact same thing in two different ways. I can definitely make this more clear in a future PR (or even just not slice it and have that state value be exactly what I want it to be in the first place!)